crates/prqlc/Cargo.toml

```diff
@@ -5,9 +5,12 @@ name = "prqlc"
 edition.workspace = true
 license.workspace = true
 repository.workspace = true
-rust-version = "1.70.0"
+rust-version.workspace = true
```

Comment:
I'm not sure if it's worth inheriting rust-version here.

Reply:
No strong view; it seemed more consistent...